docs/sources/developers/plugins/create-a-grafana-plugin/extend-a-plugin/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Extend a Grafana plugin
+menuTitle: Extend a plugin
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - development
+  - extension
+  - documentation
+description: An index of how-to topics for extending or enhancing Grafana plugins.
+weight: 200
+---
+
+# Extend a Grafana plugin
```

Comment:
This is missing `extend-the-grafana-ui-with-links`.

Reply:
Thank you, fixed.